sdks/java/io/snowflake/build.gradle

```diff
@@ -27,19 +28,17 @@ ext.summary = "IO to read and write on Snowflake."
 dependencies {
   compile library.java.vendored_guava_26_0_jre
   compile project(path: ":sdks:java:core", configuration: "shadow")
-  compile project(path: ":sdks:java:extensions:google-cloud-platform-core")
```

Comment:
All of the other changes look good, however I am suspicious of this one. @tysonjh can you find a reviewer who knows more about Snowflake IO to look at this?

Reply:
On second thought, let's just leave this as compile + permitUnusedDeclared to be safe.